rust/phoenix-channel/src/lib.rs

```diff
@@ -451,7 +454,11 @@ where
                     self.reconnect_on_transient_error(InternalError::WebSocket(e));
                     continue;
                 }
-                _ => (),
+                Poll::Ready(None) => {
```

Comment:
I think this might be another potential bug that we previously didn't check.

Reply:
👍